charts/kubefed/values.yaml

```diff
@@ -13,8 +13,8 @@ controllermanager:
   imagePullPolicy: IfNotPresent
   resources:
     limits:
-      cpu: 100m
-      memory: 128Mi
+      cpu: 400m
```

Comment:
Please, raise it to 500m, we have this limit for the CPU

Reply:
fixed